### DISCUSSION TITLE
脱离K8S在docker中通过脚本部署失败，卡在Running阶段

### DISCUSSION THREAD

**Discussion by SimleXin** [higress-group/higress#868] [Q&A] [answered]
nacos 2.3.1版本  centos7.9
脚本curl -fsSL https://higress.io/standalone/get-higress.sh | bash -s -- -a -c nacos://192.168.88.124:8848
一直卡在这里
![image](https://github.com/alibaba/higress/assets/51394686/3d5cc63b-103a-4a79-8cd8-0d4d35287860)

完整过程
![image](https://github.com/alibaba/higress/assets/51394686/dfafbb01-7ffa-4ce8-9af7-f616d11faa38)
![image](https://github.com/alibaba/higress/assets/51394686/48494a94-d000-45f9-b8ae-0309dad97806)

**Comment by SimleXin** [comment]
higress-prepare-1  这个容器一直跑不起来

  Checking config mcpbridges.default in namespace higress-system failed with 500
Checking config storage configurations...
  Checking config mcpbridges.default in namespace higress-system failed with 500
Checking config storage configurations...
  Checking config mcpbridges.default in namespace higress-system failed with 500
Checking config storage configurations...
  Checking config mcpbridges.default in namespace higress-system failed with 500
Checking config storage configurations...
  Checking config mcpbridges.default in namespace higress-system failed with 500

**Comment by CH3CHO** [comment]
看一下 apiserver 的日志

| |
Kent Dong
|
|
***@***.***
|
---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年03月12日 17:33 |
| 收件人 | alibaba/higress ***@***.***> |
| 抄送人 | Subscribed ***@***.***> |
| 主题 | Re: [alibaba/higress] 脱离K8S在docker中通过脚本部署失败，卡在Running阶段 (Discussion #868) |

higress-prepare-1 这个容器一直跑不起来

Checking config mcpbridges.default in namespace higress-system failed with 500
Checking config storage configurations...
Checking config mcpbridges.default in namespace higress-system failed with 500
Checking config storage configurations...
Checking config mcpbridges.default in namespace higress-system failed with 500
Checking config storage configurations...
Checking config mcpbridges.default in namespace higress-system failed with 500
Checking config storage configurations...
Checking config mcpbridges.default in namespace higress-system failed with 500

**Comment by SimleXin** [comment, marked as answer]
W0312 09:42:58.810666       1 logging.go:59] [core] [Channel #267 SubChannel #268] grpc: addrConn.createTransport failed to connect to {
  "Addr": "192.168.88.124:9848",
  "ServerName": "192.168.88.124:9848",
  "Attributes": null,
  "BalancerAttributes": null,
  "Type": 0,
  "Metadata": null
}. Err: connection error: desc = "transport: Error while dialing: dial tcp 192.168.88.124:9848: connect: connection refused"
W0312 09:43:03.812345       1 logging.go:59] [core] [Channel #269 SubChannel #270] grpc: addrConn.createTransport failed to connect to {
  "Addr": "192.168.88.124:9848",
  "ServerName": "192.168.88.124:9848",
  "Attributes": null,
  "BalancerAttributes": null,
  "Type": 0,
  "Metadata": null
}. Err: connection error: desc = "transport: Error while dialing: dial tcp 192.168.88.124:9848: connect: connection refused"


是因为docker-nacos没有映射gRPC的端口么

**Comment by CH3CHO** [comment]
嗯，加一下试试

| |
Kent Dong
|
|
***@***.***
|
---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年03月12日 17:45 |
| 收件人 | alibaba/higress ***@***.***> |
| 抄送人 | Kent ***@***.***>,
Comment ***@***.***> |
| 主题 | Re: [alibaba/higress] 脱离K8S在docker中通过脚本部署失败，卡在Running阶段 (Discussion #868) |

W0312 09:42:58.810666 1 logging.go:59] [core] [Channel #267 SubChannel #268] grpc: addrConn.createTransport failed to connect to {
"Addr": "192.168.88.124:9848",
"ServerName": "192.168.88.124:9848",
"Attributes": null,
"BalancerAttributes": null,
"Type": 0,
"Metadata": null
}. Err: connection error: desc = "transport: Error while dialing: dial tcp 192.168.88.124:9848: connect: connection refused"
W0312 09:43:03.812345 1 logging.go:59] [core] [Channel #269 SubChannel #270] grpc: addrConn.createTransport failed to connect to {
"Addr": "192.168.88.124:9848",
"ServerName": "192.168.88.124:9848",
"Attributes": null,
"BalancerAttributes": null,
"Type": 0,
"Metadata": null
}. Err: connection error: desc = "transport: Error while dialing: dial tcp 192.168.88.124:9848: connect: connection refused"

是因为docker-nacos没有映射gRPC的端口么

**Comment by SimleXin** [comment]
解决 开心~

**Comment by MaYuHan1234** [comment]
![image](https://github.com/user-attachments/assets/02f1daa2-34a2-4e16-9104-423e6dfd8269)

这个一直卡在这里怎么回事，解决不了，求大佬帮助！！！

我的nacos客户端显示：
![image](https://github.com/user-attachments/assets/f9b75a03-b264-449a-a557-a41d6472ad39)

docker logs查看higress-console-1的日志显示：
![image](https://github.com/user-attachments/assets/7068cff1-f116-4588-ae78-5521f6a9cc82)

望回复，感谢！！！

**Reply by CH3CHO** [reply]
还是看一下 API Server 的日志。有没有在已经装过 Higress 的 nacos 上做覆盖安装呢？

**Comment by MaYuHan1234** [comment]
体验真的太差了，早上重装了虚拟机重新安装docker环境才弄好，就配了俩路由用了一会了，下午就又访问不到了，根本就没再操作过环境了，真的想不到哪里又出了问题，又得在去排查，真TM难用，不针对你谢谢。
----- 原始邮件 -----
发件人：Kent Dong ***@***.***>
收件人：alibaba/higress ***@***.***>
抄送人：MaYuHan1234 ***@***.***>,  Comment ***@***.***>
主题：Re:_[alibaba/higress]_脱离K8S在docker中通过脚本部署失败，卡在Running阶段_(Discussion_#868)
日期：2025年03月12日 09点56分


还是看一下 API Server 的日志。有没有在已经装过 Higress 的 nacos 上做覆盖安装呢？

**Reply by johnlanni** [reply]
}. Err: connection error: desc = "transport: Error while dialing: dial tcp 192.168.88.124:9848: connect: connection refused"

nacos也是本地部署的吗，有没有可能是局域网ip变了，nacos的地址要改一下？
